aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -59,6 +59,11 @@ Parameters:
     Default: 'true'
     AllowedValues: ['false', 'true']
     Description: 'Enable Sagemaker Notebooks'
+  BatchType:
+    Type: String
+    Default: 'ec2'
+    AllowedValues: ['ec2', 'fargate']
+    Description: 'AWS Batch Compute Type'
```

Comment:
We can update the comment in L:28,35&42 to `... for EC2 Batch Compute Environment (ignored for Fargate)`

Reply:
Updated for Min and Desired.  Left Max as-is per the earlier comment.